### PR TITLE
Pipe cleaner cable color fix

### PR DESCRIPTION
**Pull request by Arkatos1**
## About The Pull Request

This PR fixes a bug where service cyborgs were unable to set a color of the pipe cleaner cable they possess due to not setting an actual color code for a cable colorization and a duplicit name variable.

Also adds an option for cancelling color choosing input as per maintainer suggestion.

## Why It's Good For The Game

Bugfix.

## Changelog
:cl: Arkatos
fix: Service cyborgs are now able to properly set a pipe cleaner cable color for their art needs.
tweak: Service cyborgs can now cancel color choosing input for pipe cleaner cables in case they have changed their minds about it.
/:cl:

### CODE REVIEW

File: code/modules/power/pipecleaners.dm [hunk 1]
```diff
@@ -203,15 +202,17 @@ By design, d1 is the smallest direction and d2 is the highest
 	full_w_class = WEIGHT_CLASS_SMALL
 	grind_results = list("copper" = 2) //2 copper per pipe_cleaner in the coil
 	usesound = 'sound/items/deconstruct.ogg'
+	/// Currently set cable color
+	var/pipe_cleaner_color = "red"
```
Comment (Rohesie) [inline code/modules/power/pipecleaners.dm:206]:
```suggestion
	var/pipe_cleaner_color = COLOR_RED
```

File: code/modules/power/pipecleaners.dm [hunk 2]
```diff
@@ -203,15 +202,17 @@ By design, d1 is the smallest direction and d2 is the highest
...
 
 /obj/item/stack/pipe_cleaner_coil/cyborg
 	is_cyborg = 1
 	custom_materials = null
 	cost = 1
 
 /obj/item/stack/pipe_cleaner_coil/cyborg/attack_self(mob/user)
-	var/pipe_cleaner_color = input(user,"Pick a pipe cleaner color.","Cable Color") in sortList(list("red","yellow","green","blue","pink","orange","cyan","white"))
-	pipe_cleaner_color = pipe_cleaner_color
+	var/selected_color = input(user, "Pick a pipe cleaner color.", "Cable Color") in sortList(list("red", "yellow", "green", "blue", "pink", "orange", "cyan", "white"))
```
Comment (Rohesie) [inline code/modules/power/pipecleaners.dm:214]:
```suggestion
	var/selected_color = input(user, "Pick a pipe cleaner color.", "Cable Color") in list("red", "yellow", "green", "blue", "pink", "orange", "cyan", "white")
```

Comment (Rohesie) [reply code/modules/power/pipecleaners.dm:214]: adding a `null` provision for if the person chooses to cancel the event would be nice as well

**Comment by Arkatos1** [comment]
Addressed all reviews and added a null provision as per maintainer suggestion, so service cyborgs can cancel color choosing input if change their minds about it.

**Review by Rohesie — APPROVED** [review]

**Pull request merged**